lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -459,17 +443,22 @@ def _update_usage_from_flow_stat(self, current_usage, flow_stat):
         # use a compound key to separate flows for the same rule but for
         # different subscribers
         key = sid + "|" + rule_id
+
         if ipv4_addr:
             key += "|" + ipv4_addr
         elif ipv6_addr:
             key += "|" + ipv6_addr
-        record = current_usage[key]
-        record.rule_id = rule_id
-        record.sid = sid
 
         rule_version = _get_version(flow_stat)
         if not rule_version:
             rule_version = 0
+
+        key += "|" + str(rule_version)
+
+        record = current_usage[key]
+        record.rule_id = rule_id
+        record.sid = sid
+
```

Comment:
do we still need map here? list of stats record enough, no?

Reply:
multiple flows can match to one record (usually at least 2 flows in the policyrule flowlist) so we still need map